iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/LoadingLimitsUtil.java

```diff
@@ -96,6 +96,9 @@ public static <L extends LoadingLimits, A extends LoadingLimitsAdder<L, A>> void
       * @param limits the limits to copy
       */
     public static <L extends LoadingLimits, A extends LoadingLimitsAdder<L, A>> A initializeFromLoadingLimits(A adder, L limits) {
+        if (limits == null) {
+            throw new PowsyblException("Cannot initialize new limits from null limits");
+        }
```

Comment:
I think in the end that it is better to just warn the user (using a logger) and return an empty adder than to stop everything with a throw